react-ui-kit/src/PaginationSimple/index.tsx

```diff
@@ -25,9 +25,13 @@ const KEY_CODE_DOT = 190;
 const KEY_CODE_NUM_DOT = 110;
 
 export interface IPaginationSimpleProps {
+  /** Текущая страница */
   currentPage: number;
+  /** Список задизейбленных страниц */
```

Comment:
```suggestion
  /** Список отключенных страниц */
```
Мне кажется так более читаемо :)

Reply:
Саш, я уже влила пул реквест в мастер, так как был апрув. Могу поправить это уже в рамках другого пул реквеста